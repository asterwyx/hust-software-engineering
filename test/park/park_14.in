preset user AQ
preset map 1 A 0
preset map 2 A 0
preset map 3 Q 0
preset map 5 Q 0
preset gift A robot 5
preset gift A barrier 5
preset gift Q robot 5
preset gift Q barrier 5
preset userloc A 12 0
preset userloc Q 45 0
preset nextuser Q
step 4 #Q:45-49 park
step 2 #A:12->14 park
robot
block 1
block 5
block -1
block -8
robot
sell 3
step 1 #Q:49->50
n
block 1
block 2
block -1
robot
sell 1
step 1 #A:14->15
n
dump