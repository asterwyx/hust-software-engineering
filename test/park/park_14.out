user AQ
map 2 A 0
fund A 10400
credit A 0
userloc A 15 0
gift A barrier 2
gift A robot 4
map 5 Q 0
fund Q 10400
credit Q 0
userloc Q 50 0
gift Q barrier 1
gift Q robot 3
barrier 13
barrier 41
barrier 48
nextuser Q
